ext/opcache/zend_shared_alloc.c

```diff
@@ -612,7 +612,7 @@ const char *zend_accel_get_shared_model(void)
 	return g_shared_model;
 }
 
-void zend_accel_shared_protect(bool protected)
+void zend_accel_shared_protect(int mode)
```

Comment:
should not be reverted

Reply:
Here, I'm not completely sure. It's not a problem to switch to ``bool``, but using ``int`` we may extend the same function to use READ/WRITE/EXEC protection.